content/english/java-basics/activity-1.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Print Statements and Comments"
+description: "Introduce println() methods and comments in Java."
+date: 2020-07-08T00:00:00Z
+---
+
+### 1. Let the Program Speak! 
+
+Let’s try to print `Hello World` out by typing `System.out.print("Hello World");` 
+
```

Comment:
A page that covers the basics of using Repl.it would be helpful. You can copy-paste most of it from our other existing workshops (but this can be done separately).